invesalius/navigation/mtms.py

```diff
@@ -0,0 +1,115 @@
+import win32com.client
+
+import numpy as np
+import pandas as pd
+import random
+import time
+
+import invesalius.data.coregistration as dcr
+
+class mTMS():
```

Comment:
It would be better to open a dialog for the user to select where the connection is. Because now this is hard-coded. Something like we do for the tracking device markers of NDI. The user do it once, and the path is saved in the .config file. If not now, make a note for future adjustement.

Reply:
I agree! I will put this as TODO.